pkgs/development/python-modules/atomman/default.nix

```diff
@@ -56,13 +65,18 @@ buildPythonPackage rec {
   ];
 
   checkPhase = ''
-    # pytestCheckHook doesn't work
-    pytest tests -k "not test_rootdir and not test_version \
-      and not test_atomic_mass and not imageflags \
-      and not test_build_unit and not test_set_and_get_in_units \
-      and not test_set_literal and not test_scalar_model " \
-      --ignore tests/plot/test_interpolate.py \
-      --ignore tests/tools/test_vect_angle.py
+    runHook preCheck
+
+    # pytestCheckHook doesn't work because it imports atomman from the current
+    # directory instead of from where `pip` installs it, and it cannot import
```

Comment:
Did you try rm'ing that directory or prepending PYTHONPATH with the directory from $out? If that works we prefer that route.

Reply:
I got it to work by removing the directory. Thank you for the suggestion and, as always, your hard work.